### PR TITLE
Add ci and others v2

### PR DESCRIPTION
**Pull request by gabriel-tessier**

### CODE REVIEW

**Comment by gabriel-tessier** [comment]
@tb102122 

The previous PR had too much conflicts I created a new branch from master and cherry pick your changes.
Can you review the commit and confirm that no code was lost.

File: .github/ISSUE_TEMPLATE/bug_report.md
Comment (gabriel-tessier) [inline .github/ISSUE_TEMPLATE/bug_report.md:30]: Do we need smartphone section? The lib will be only installed on PC I don't think people will install it on smartphone.

File: .github/ISSUE_TEMPLATE/bug_report.md
Comment (gabriel-tessier) [inline .github/ISSUE_TEMPLATE/bug_report.md:28]: What is browser for?

What about a more detailed Version section: 
- Python version [e.g. 3.11]
- Graphviz version [e.g. 11.0.0]

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (gabriel-tessier) [inline .github/PULL_REQUEST_TEMPLATE.md:6]: This section I don't like it feel like the contributor will provide his work and we (diagrams-web) can copy and redistribute it with a commercial license... 

Better to remove it, as the source code use a MIT license it should imply that the contribution will be under this license.

File: .github/ISSUE_TEMPLATE/bug_report.md
Comment (tb102122) [reply .github/ISSUE_TEMPLATE/bug_report.md:30]: yes we cam remove it

File: .github/ISSUE_TEMPLATE/bug_report.md
Comment (tb102122) [reply .github/ISSUE_TEMPLATE/bug_report.md:28]: yes makes sense to add that

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (tb102122) [reply .github/PULL_REQUEST_TEMPLATE.md:6]: Your call I am fine with it

Comment (tb102122) [reply .github/PULL_REQUEST_TEMPLATE.md:6]: do you like me to do some of these changes or will you take care?

Comment (gabriel-tessier) [reply .github/PULL_REQUEST_TEMPLATE.md:6]: Go ahead, please.

**Comment by gabriel-tessier** [comment]
@tb102122 
Can you check also for what you mention in the Issue https://github.com/diagrams-web/diagrams-xtd/issues/2:

> I did one important change in the project.toml to support 3.12 and remove 3.7 support please migrate that as well

File: .github/workflows/py-publish.yml
Comment (tb102122) [inline .github/workflows/py-publish.yml:26]: this still needs to be updated @gabriel-tessier

**Review by tb102122 — APPROVED** [review]
do you like to remove the publishing step or keep it?

File: .github/workflows/py-publish.yml
Comment (gabriel-tessier) [reply .github/workflows/py-publish.yml:26]: @tb102122 
I submitted an organization called diagrams-web on pypi it's still in pending state from more than 2 weeks.
I didn't want to publish it on my own account and transfer it later as I want to have the package on the organization from the beginning.


**diagrams-web Community Request Submitted**

**Submitted Jul 13, 2024**

 
But as it take more time than I expected for Pypi to approve, I will publish it on my own account.
About the py-publish.yml, let's keep this as it and wait to have the information for the organization to set teh secrets.

You will receive an invitation for the organization as soon as it get approved.

**Pull request merged**
